README.md

```diff
@@ -0,0 +1,54 @@
+# RHVoice
+
+RHVoice is a free and open source speech synthesizer.
+
+## Features
+
+### Speech synthesis method
+
+RHVoice uses statistical parametric synthesis. It relies on existing
+open source speech technologies (mainly
+[HTS](http://hts.sp.nitech.ac.jp) and related software).
+
+Voices are built from recordings of natural speech. They have small
+footprints, because only statistical models are stored on users'
+computers. And though the voices lack the naturalness of the
+synthesizers which generate speech by combining segments of the
+recordings themselves, they are still very intelligible and resemble
+the speakers who recorded the source material.
+
+### Supported languages
+
+Initially, RHVoice could speak only Russian. Now it also supports:
+
+* American English
+* Brazilian Portuguese
+* Esperanto
+* Georgian
+* Ukrainian
+* Kyrgyz
+* Tatar
+
+In theory, it is possible to implement support for
+other languages, if all the necessary resources can be found or
+created.
+
+### Supported platforms
+
+RHVoice supports folowing platforms:
+
+* Windows
+* GNU/Linux
+* Android
+
+It is compatible with
+standard text-to-speech interfaces on these platforms: SAPI5 on
+Windows, [Speech Dispatcher](http://devel.freebsoft.org/speechd) on
+GNU/Linux and Android's text-to-speech APIs. It can also be used by
+the [NVDA screen reader](http://www.nvaccess.org) directly (the driver
+is provided by RHVoice itself).
+
+## license
+
+The main library is distributed under [LGPL v2.1](https://www.gnu.org/licenses/lgpl-2.1.html) or later.
+some components and voices use other licenses.for more information please read [license.md](license.md) file.
```

Comment:
The library is distributed under GPL, because it ships binaries compiled with MAGE.

Reply:
However, the sources of the library are using LGPL license.